src/pages/_app.js

```diff
@@ -7,7 +7,7 @@ import { ThemeProvider } from "@material-ui/core/styles";
 import CssBaseline from "@material-ui/core/CssBaseline";
 import { CacheProvider } from "@emotion/react";
 import createCache from "@emotion/cache";
-// import theme from "../src/theme";
+import theme from "../utils/theme";
```

Comment:
@chougulesurajb  commented this... hope you are not undoing his change?

Reply:
Im adding theme to app so i have created theme file. Thanks

Reply:
No worries I was adding theme file but got some error. @SaiCharanTejR If you have resolved those. then we can take in. thanks